src/locale/locale.js

```diff
@@ -0,0 +1,48 @@
+export default class Locale {
```

Comment:
the "normal" way for translations is to use Mautic.translate(). Can you use that here?

Reply:
done